dependencies.yml

```diff
@@ -24,10 +24,6 @@ dependencies:
     trace-agent:
       artifact: datadog/dd-java-agent-{{ version }}.jar
       version: 0.101.0
-  dynatrace:
-    agent:
-      artifact: "{{ url }}/e/{{ environment }}/api/v1/deployment/installer/agent/unix/paas/latest?include=java&bitness=64&Api-Token={{ token }}"
-      managed: false
```

Comment:
As we're removing the old behavior entirely, it will break for the existing DynaTrace customers. Should we add some backwards compatibility ?

Reply:
I won't expect integration to be broken for existing customers since the same environment variables would be enough for the new approach as well. The names of metrics and the scope can change though. Still, I'd vote to remove old behavior because:

* It's not supported anymore
* It may cause more confusion in the future if we have 2 different ways of integration
* We were not supporting Dynatrace officially

Reply:
Sounds good to me. Just need to keep support in the loop.